common/types/src/main/java/io/helidon/common/types/DefaultAnnotationAndValue.java

```diff
@@ -227,7 +227,7 @@ public DefaultAnnotationAndValue build() {
         public Builder typeName(TypeName val) {
             Objects.requireNonNull(val);
             this.typeName = val;
-            return this;
+            return identity();
```

Comment:
This is not needed, as you are not returning a generic. If you use this "hackish" solution, no change required here.

Reply:
pushed